backend/src/schema/resolvers/shout.spec.js

```diff
@@ -102,6 +102,22 @@ describe('shout and unshout posts', () => {
         })
       })
 
+      it('adds `createdAt` to `SHOUT` relationship', async () => {
+        variables = { id: 'another-user-post-id' }
+        await mutate({ mutation: mutationShoutPost, variables })
+        const relation = await instance.cypher(
+          'MATCH (user:User {id: $userId1})-[relationship:SHOUTED]->(node {id: $userId2}) WHERE relationship.createdAt IS NOT NULL RETURN relationship',
+          {
+            userId1: 'current-user-id',
+            userId2: 'another-user-post-id',
+          },
+        )
+        const relationshipProperties = relation.records.map(
+          record => record.get('relationship').properties.createdAt,
+        )
+        expect(relationshipProperties[0]).toEqual(expect.any(String))
```

Comment:
> [<img alt="mattwr18" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/mattwr18) **Authored by [mattwr18](https://github.com/mattwr18)**
_<time datetime="2019-10-18T13:53:26Z" title="Friday, October 18th 2019, 3:53:26 pm +02:00">Oct 18, 2019</time>_

---
> **Outdated (history rewrite)** - original diff
---
```diff
@@ -100,6 +100,18 @@ describe('shout and unshout posts', () => {
           data: { Post: [{ id: 'another-user-post-id', shoutedBy: [{ id: 'current-user-id' }] }] },
           errors: undefined,
         })
+        // Test to make sure SHOUT relationship has "createdAt" date.
+        const relation = await instance.cypher(
+          'MATCH (user:User {id: $userId1})-[relationship:SHOUTED]->(node {id: $userId2}) WHERE relationship.createdAt IS NOT NULL RETURN relationship',
+          {
+            userId1: 'current-user-id',
+            userId2: 'another-user-post-id',
+          },
+        )
+        const relationshipProperties = relation.records.map(
+          record => record.get('relationship').properties.createdAt,
+        )
+        expect(relationshipProperties[0]).toEqual(expect.any(String))
```


same as above... you can have an `it` block where you add a description for the test instead of a comment
then run `await mutate({ mutation: mutationShoutPost, variables })`
then this code you added

Reply:
> [<img alt="roschaefer" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/roschaefer) **Authored by [roschaefer](https://github.com/roschaefer)**
_<time datetime="2019-10-18T13:54:23Z" title="Friday, October 18th 2019, 3:54:23 pm +02:00">Oct 18, 2019</time>_

---


Same as above :point_up: in this test case we already have two assertions - because of a weakness in the `shout` resolver: Instead of returning the shouted post, we return `Boolean`. That's why we have to send another graphql query just to make sure it really got shouted by the current user.

Reply:
> [<img alt="KapilJ22" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/KapilJ22) **Authored by [KapilJ22](https://github.com/KapilJ22)**
_<time datetime="2019-10-20T19:24:10Z" title="Sunday, October 20th 2019, 9:24:10 pm +02:00">Oct 20, 2019</time>_

---


Thanks again for review @mattwr18 . it's done as well.

Reply:
> [<img alt="KapilJ22" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/KapilJ22) **Authored by [KapilJ22](https://github.com/KapilJ22)**
_<time datetime="2019-10-20T19:25:19Z" title="Sunday, October 20th 2019, 9:25:19 pm +02:00">Oct 20, 2019</time>_

---


Thanks @roschaefer for review. I made changes as requested.